inc/ticket.class.php

```diff
@@ -4184,6 +4184,15 @@ function showForm($ID, $options = []) {
          }
       }
 
+      // Restore saved value and override $this->fields
+      if (count($saved)) {
+         foreach ($saved as $name => $value) {
+            if (isset($this->fields[$name])) {
+               $this->fields[$name] = $saved[$name];
+            }
+         }
+      }
```

Comment:
Maybe this piece of code, from line 4177 to 4194, should be mutualized with Change and Problem classe. See #8771 where I add to add save input restoration process on these classes.

Reply:
ok, so it is now possible to restore the values for changes and problems
done